src/app/activity-data/models/activity-definition.model.ts

```diff
@@ -2,18 +2,21 @@ import {QuestionDefinition} from "./question-definition.model";
 
 export class ActivityDefinition {
 
-  constructor( public activityCode: string, public activityName: string, public activityVersion: string, public questions: Array<QuestionDefinition> ) {
+  constructor( public activityCode: string, public activityName: string, public activityVersion: string, public questions: QuestionDefinition[],
+               public displayOrder:number=1, public showActivityStatus = true) {
```

Comment:
can we get `displayOrder:number` and showActivityStatus from activity_definition now?